eulerpi/core/model.py

```diff
@@ -96,6 +96,21 @@ def forward_and_jacobian(
 
         return self.forward(param), self.jacobian(param)
 
+    def param_is_within_domain(self, param: np.ndarray) -> bool:
+        """Checks whether a parameter is within the parameter domain of the model.
+        Overwrite this function if your model has a more complex parameter domain than a box.
+
+        Args:
+            param(np.ndarray): The parameter to check.
+
+        Returns:
+            bool: True if the parameter is within the limits.
+
+        """
+        return np.all(param >= self.param_limits[:, 0]) and np.all(
```

Comment:
```suggestion
               return True
```

The limits are already checked in `eulerpi/core/transformations.py`.
PS: I couldn't find a way to suggest changes spanning multiple rows.